DEV_ENVIRONMENT.md

```diff
@@ -0,0 +1,22 @@
+# Developer guide: getting your environment set up
+
+1. Make sure you have `node` installed with a version at _least_ 4.2.3.
+2. Run `npm install -g angular-cli` to install the Angular CLI.
```

Comment:
I figure we tell people to use the cli directly because that is meant to be the standard for angular apps, as opposed to whatever custom npm scripts individual projects might add.